src/hasher.cpp

```diff
@@ -77,14 +49,6 @@ namespace libtorrent
 #elif TORRENT_USE_COMMONCRYPTO
 		CC_SHA1_Init(&m_context);
 #elif TORRENT_USE_CRYPTOAPI
```

Comment:
this preprocessor statement is pointless now, since there's no code in here. you might as well remove it